eng/icu.mk

```diff
@@ -43,11 +43,13 @@ include icu.$(TARGET_OS).mk
 
 UNAME_S := $(shell uname -s)
 LINKER_OVERRIDE = 
+ifeq ($(filter $(USE_DEFAULT_LINKER),false),)
 ifeq ($(UNAME_S),Linux)
 ifeq ($(TARGET_OS),browser)
 	LINKER_OVERRIDE = LDFLAGS=-fuse-ld=lld
```

Comment:
maybe we can check whether `lld` exists and only use it if it does?

Reply:
Good idea, let's see